backend/code_review_backend/issues/api.py

```diff
@@ -152,6 +152,9 @@ class IssueCheckStats(generics.ListAPIView):
     queryset = (
         Issue.objects.values("analyzer", "check", "diff__revision__repository__slug")
         .annotate(total=Count("id"))
+        .annotate(
+            publishable=Count("id", filter=Q(in_patch=True) & Q(new_for_revision=True))
```

Comment:
Since we have this info in the bot, maybe we could make the bot upload it?

Reply:
I think it makes more sense if the backend has all the pieces to decide what's publishable, save that decision, and give it to the bot (which becomes dumb and publish when the backend says so)

Reply:
Agreed, that's what we have to do in the end for before/after. We could move the publishable decision in the backend now, as a first step (so we also avoid this duplicity). We can do it in a separate PR though.

Reply:
Filed that https://github.com/mozilla/code-review/issues/390